src/test/java/org/jabref/logic/shared/DBMSProcessorTest.java

```diff
@@ -164,18 +164,46 @@ void testUpdateEqualEntry(DBMSType dbmsType, DBMSConnection dbmsConnection, DBMS
 
     @ParameterizedTest
     @MethodSource("org.jabref.logic.shared.TestManager#getTestingDatabaseSystems")
-    void testRemoveEntry(DBMSType dbmsType, DBMSConnection dbmsConnection, DBMSProcessor dbmsProcessor) throws SQLException {
```

Comment:
Can be keep the single test? -> Meaning the additional test for two entries is OK, but we should also keep the simple (and probably always succeeding test). - Reason: The for loop above is not entered for one entry and thus this test case is special.

Reply:
I could, but it would still have to be `dbmsProcessor.removeEntries(Collections.singletonList(entryToRemove))` because I got rid of `DBMSProcessor.removeEntry` in the singular. Is it still worth it?

Reply:
Sure. The tests should be independent of the implemention somehow. Tests should cover the general case, but also edge cases, null list, 0 element list, list with 1 elements, list with integer.max elements (which is impossible IMHO), but the other cases should be easy to do?

Reply:
Okay, in that case I will use `Collections.singletonList`. Should I add a test for each of those edge cases as well?

Reply:
Sure, please go ahead. In case tests are easy to write, they should be written. - The thing is: The implementation could change in 1 year (who knows) and the test will still cover that case.

Please also add a test for

- `dbmsProcessor.removeEntries(Collections.emptyList())`
- `dbmsProcessor.removeEntries(null)` (should throw a NPE)

Reply:
Throw on empty list? If so, throw what? I can't imagine why someone would want to call `removeEntries` on purpose on an empty list.